components/esp_lvgl_port/src/lvgl9/esp_lvgl_port_disp.c

```diff
@@ -428,7 +428,7 @@ static void _lvgl_port_transform_monochrome(lv_display_t *display, const lv_area
     }
 }
 
-void lv_display_rotate_area(lv_display_t *disp, lv_area_t *area)
+void lvgl_port_rotate_area(lv_display_t *disp, lv_area_t *area)
```

Comment:
Other option for consideration:
* We can declare this function as `weak` so the linker can use either this or LVGL's implementation, if it is present

I'm not sure though, whether this is a good idea for this case...

Reply:
I suggest, we can keep it like this now.